Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -544,18 +544,18 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
             var commandLineLogMessage = process.StartInfo.FileName + " " + process.StartInfo.Arguments;
             _logger.LogInformation(commandLineLogMessage);
 
-            var logFilePrefix = "ffmpeg-transcode";
+            var logFilePrefix = "FFmpeg.Transcode-";
             if (state.VideoRequest != null
                 && EncodingHelper.IsCopyCodec(state.OutputVideoCodec))
             {
                 logFilePrefix = EncodingHelper.IsCopyCodec(state.OutputAudioCodec)
-                    ? "ffmpeg-remux"
-                    : "ffmpeg-directstream";
+                    ? "FFmpeg.Remux-"
+                    : "FFmpeg.DirectStream-";
             }
 
-            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + "-" + Guid.NewGuid() + ".txt");
+            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + DateTime.Now.ToString("yyyy-MM-dd_HH-mm-ss", CultureInfo.InvariantCulture) + "_" + state.Request.MediaSourceId + "_" + Guid.NewGuid().ToString("N").Substring(0, 8) + ".log");
```

Comment:
```suggestion
            var logFilePath = Path.Combine(
                _serverConfigurationManager.ApplicationPaths.LogDirectoryPath,
                $"{logFilePrefix}{DateTime.Now:yyyy-MM-dd_HH-mm-ss}_{state.Request.MediaSourceId}_{Guid.NewGuid().ToString()[..8]}.log");
```

Shorter way of doing the same thing